pages/odontologica.js

```diff
@@ -4,19 +4,19 @@ import BannerCta from '../components/Banner/BannerCta';
 import Feature from '../components/Feature/Feature';
 import Carousel from '../components/Carousel';
 import ProductSlider from '../components/Product/ProductSlider';
-
 import GeneratedVideos from '../components/Videos/Videos'
 import MockedFeatures from '../mocks/odontologicaFeatures';
+import  Data  from '../mocks/odontologicaMock';
```

Comment:
Esto lo podes importar de la siguiente forma:
```suggestion
import  { productCover, appCover, features }  from '../mocks/odontologica';
```